PrReviewCode/Program.cs

```diff
@@ -4,6 +4,8 @@ public class Program
 {
     public static void Main(string[] args)
     {
+        throw new Exception("Test Exception");
```

Comment:
Uncaught exception in Main method will crash the application. Ensure this is only present in test environments or add proper error handling.

 **SUGGESTION:**

```suggestion
 // Add conditional check for test environment before throwing
```